packages/shared/src/components/image/VideoImage.tsx

```diff
@@ -18,7 +18,7 @@ const VideoImage = ({
     <div
       className={classNames(
         wrapperClassName,
-        'relative flex h-auto w-full items-center justify-center rounded-12',
+        'relative flex h-auto items-center justify-center rounded-12',
```

Comment:
@nensidosari this fixes the video thumbnail sizing issue as well, it also does not affect the previous layout cards based on my testing, so we should not need to copy this component to `v1` just for this change.

Reply:
Ah, got the issue, it messes with the video types on shared posts

Reply:
We'll need to fix @DragosIorgulescu

Reply:
updated so we only apply width adjustments on the new layout